doc/qbk/release_notes.qbk

```diff
@@ -11,6 +11,68 @@
 
 [/-----------------------------------------------------------------------------]
 
+[heading Boost 1.73]
+
+[*API Changes]
+
+* Nested const and mutable buffer types for all
+  Beast dynamic buffers are refactored. Affected types:
+  - buffers_adapter
```

Comment:
need backticks for identifiers, e.g. `buffers_adaptor` (note spelling on this one)